calico-enterprise_versioned_docs/version-3.15/releases.json

```diff
@@ -1,4 +1,257 @@
 [
+  {
+    "title": "v3.15.4",
+    "tigera-operator": {
+      "image": "tigera/operator",
+      "version": "v1.28.12",
```

Comment:
```suggestion
      "version": "v1.28.14",
```

Reply:
@danudey  does my suggestion look correct ?

Reply:
Fixed.

Reply:
I don't see the fixed commit. @bmckercher123

Reply:
I rebased and pushed but PR doesn't seem to be updating.